integrations/pagerduty/config.yaml

```diff
@@ -0,0 +1,18 @@
+# This is an example configuration file for the integration service.
+# Please copy this file to config.yaml file in the integration folder and edit it to your needs.
+initializePortResources: true
+port:
+  clientId: {{ from env PORT_CLIENT_ID }} # Can be loaded via environment variable: PORT_CLIENT_ID
+  clientSecret: {{ from env PORT_CLIENT_SECRET }} # Can be loaded via environment variable: PORT_CLIENT_SECRET
+# The event listener to use for the integration service.
+eventListener:
+  type: POLLING
+integration:
+  # The identifier of this integration instance.
+  identifier: pagerduty-exporter
+  # The type of the integration.
+  type: "pagerduty"
+  config:
+    token: {{ from env PAGERDUTY_TOKEN }}
+    app_url: https://your_app_url.com
```

Comment:
use this syntax `{{ from env PAGERDUTY_TOKEN }}` for the other configurable parameters